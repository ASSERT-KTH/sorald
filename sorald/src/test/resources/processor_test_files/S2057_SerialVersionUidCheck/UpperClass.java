// Test for rule s2057
import java.io.Serializable;

class UpperClass implements Serializable { // Noncompliant
}
