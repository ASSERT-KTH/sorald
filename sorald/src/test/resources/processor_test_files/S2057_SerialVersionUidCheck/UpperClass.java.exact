// Test for rule s2057
import java.io.Serializable;

class UpperClass implements Serializable {
###
	private static final long serialVersionUID = 1L;
###
}
