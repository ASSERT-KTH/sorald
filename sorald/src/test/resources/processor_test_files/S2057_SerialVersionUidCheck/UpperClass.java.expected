// Test for rule s2057
import java.io.Serializable;

class UpperClass implements Serializable { // Noncompliant
  private static final long serialVersionUID = 1L;
}
